payload/hardware/transmitter.py

```diff
@@ -105,6 +107,14 @@ def _send_message_worker(self, message: str) -> None:
 
             time.sleep(5)  # Keep the pin low for the transmission duration
 
+        self._pull_pin_high()  # Deactivate PTT via GPIO pin pull-up
```

Comment:
I'm going to check if there's a way to do this with gpiozero.. i don't think there might be tho